examples/stream.ru

```diff
@@ -8,17 +8,20 @@
 #   puma stream.ru                # gem install puma
 
 require 'sinatra/base'
+require 'sinatra/stream/reel'
 
 class Stream < Sinatra::Base
   get '/' do
-    content_type :txt
+    #content_type :txt
 
-    stream do |out|
-      out << "It's gonna be legen -\n"
+    #stream do |out|
+    Rack::StreamingResponse.new do |out|
+      out.hijack!(env)
```

Comment:
@zzak no worries, but just as a sidenote, I really think that this has to be addressed first on the rack level.